equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm5Mapper.java

```diff
@@ -44,9 +44,72 @@ public interface Crm5Mapper {
+    @Mapping(target="allCosts.accruedCosts.waiting.cost", source="ctd_waiting_time_cost")
+    @Mapping(target="allCosts.accruedCosts.letters.total", source="ctd_letters")
+    @Mapping(target="allCosts.accruedCosts.letters.cost", source="ctd_letters_cost")
+    @Mapping(target="allCosts.accruedCosts.telephoneCalls.total", source="ctd_phone_calls")
+    @Mapping(target="allCosts.accruedCosts.telephoneCalls.cost", source="ctd_phone_calls_cost")
+    @Mapping(target="allCosts.accruedCosts.mileage.total", source="ctd_mileage_miles")
+    @Mapping(target="allCosts.accruedCosts.mileage.cost", source="ctd_mileage_cost")
+    @Mapping(target="allCosts.accruedCosts.otherDisbursement.cost", source="ctd_other_cost")
+    @Mapping(target="allCosts.accruedCosts.totalCost.cost", source="ctd_total_costs")
+    @Mapping(target="allCosts.anticipatedCosts.attendance.time", source="ac_attendance_time")
+    @Mapping(target="allCosts.anticipatedCosts.attendance.cost", source="ac_attendance_time_cost")
+    @Mapping(target="allCosts.anticipatedCosts.preparation.time", source="ac_preparation_time")
+    @Mapping(target="allCosts.anticipatedCosts.preparation.cost", source="ac_preparation_time_cost")
+    @Mapping(target="allCosts.anticipatedCosts.advocacy.time", source="ac_advocacy_time")
+    @Mapping(target="allCosts.anticipatedCosts.advocacy.cost", source="ac_advocacy_time_cost")
+    @Mapping(target="allCosts.anticipatedCosts.travel.time", source="ac_travel_time")
+    @Mapping(target="allCosts.anticipatedCosts.travel.cost", source="ac_travel_time_cost")
+    @Mapping(target="allCosts.anticipatedCosts.waiting.time", source="ac_waiting_time")
+    @Mapping(target="allCosts.anticipatedCosts.waiting.cost", source="ac_waiting_time_cost")
+    @Mapping(target="allCosts.anticipatedCosts.letters.total", source="ac_letters")
+    @Mapping(target="allCosts.anticipatedCosts.letters.cost", source="ac_letters_cost")
+    @Mapping(target="allCosts.anticipatedCosts.telephoneCalls.total", source="ac_phone_calls")
+    @Mapping(target="allCosts.anticipatedCosts.telephoneCalls.cost", source="ac_phone_calls_cost")
+    @Mapping(target="allCosts.anticipatedCosts.mileage.total", source="ac_mileage")
+    @Mapping(target="allCosts.anticipatedCosts.mileage.cost", source="ac_mileage_cost")
+    @Mapping(target="allCosts.anticipatedCosts.otherDisbursement.cost", source="ac_other_cost")
+    @Mapping(target="allCosts.anticipatedCosts.totalCost.cost", source="ac_total_costs")
+    @Mapping(target="caseHistory.summary", source="ch_case_history")
+    @Mapping(target="caseHistory.addtionalInfo", source="ch_additional_info")
+    @Mapping(target="solicitor.declaration.date", source="solicitor_sign_date")
+    @Mapping(target="solicitor.declaration.name", source="solicitor_sign_forename")
+    @Mapping(target="solicitor.certification.date", source="certification_sol_date")
+    @Mapping(target="solicitor.certification.name", source="certification_sol_name")
```

Comment:
I think mappings like this: `@Mapping(target="capitalDetails.isUnder18", source="client_under_18")` can be factored out like:
```
@Mapping(target="isUnder18", source="client_under_18"
CapitalDetailsDTO getCapitalDetailsDTOFromModel(Crm5DetailsModel crm5DetailsModel);
```
And not sure if the original mapping can be removed since it would be picked up automatically or replaces for something like:
`@Mapping(target="capitalDetails")`

Reply:
This way, each section will have its own mapping instead of having a 100+ lines of a single mapping as a whole

Reply:
I review with Solomon and we agree on an action plan. We are going to create a follow up ticket for refactoring. We will also try to follow this approach in the next tasks that are similar